examples/dreambooth/train_dreambooth.py

```diff
@@ -224,6 +224,30 @@ def import_model_class_from_model_name_or_path(pretrained_model_name_or_path: st
         raise ValueError(f"{model_class} is not supported.")
 
 
+def compute_snr(timesteps, noise_scheduler):
```

Comment:
Maybe we should graduate this function to `training_utils.py` at this point as Min-SNR training is quickly becoming a gold standard. @patrickvonplaten, thoughts?

Reply:
totally. especially because it's unmodified reference code

Reply:
Yeah ok for me!